reactor-netty-http/src/main/java/reactor/netty/http/QueryStringDecoder.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (c) 2012-2023 VMware, Inc. or its affiliates, All Rights Reserved.
```

Comment:
since the codes came from Netty, should the license statement be replaced with the one from Netty?

Reply:
@violetagg could you have a look and see if the PR is good to go?